app/helpers/highlight_boxes_helper.rb

```diff
@@ -0,0 +1,9 @@
+module HighlightBoxesHelper
+  def data_tracking?(items)
+    items.each do |item|
+      return true if item[:link][:data_attributes]
+    end
+
+    false
```

Comment:
Do you need false here? Presumably if there's no data attributes this method doesn't return anything.

Reply:
@andysellick If the `each` block doesn't evaluate to anything, it returns the whole `items` array back to the call. This doesn't happen with `map`, because `map` creates a brand new array and `each` doesn't.